src/styles/blocks/center.scss

```diff
@@ -0,0 +1,46 @@
+.clock {
+  position: relative;
+  width: $clock-size;
+  height: $clock-size;
```

Comment:
I think we can use mixins in cases like this

Reply:
fix.